app-libs/stf/poc/src/lib.rs

```diff
@@ -9,10 +18,10 @@ pub fn main() -> String {
     "#;
 
 	let person = from_json_string(json_string);
-	println!("{:?}", person);
+	info!("{:?}", person);
```

Comment:
`println!` is not available in `no_std`, I have made a litle trick and used the `no_std` compatible `log` crate here. In the sgx-libraries you would also find a `println`, so you can use it in the sgx-domain, but I thought this is an overkill here.